src/WebJobs.Script/Scale/ApplicationPerformanceCounters.cs

```diff
@@ -31,6 +31,14 @@ public class ApplicationPerformanceCounters
 
         public long NamedPipeLimit { get; set; }
 
+        public long RemoteDirMonitors { get; set; }
+
+        public long RemoteDirMonitorLimit { get; set; }
```

Comment:
Initially this limit will be 0 - so our monitor will noop. When the limit is enabled however, we'll apply it.